dataset/test/data_array_test.cpp

```diff
@@ -6,7 +6,7 @@
 #include "scipp/dataset/event.h"
 #include "scipp/dataset/histogram.h"
 #include "scipp/dataset/unaligned.h"
-#include "scipp/variable/comparison.h"
+#include "scipp/variable/variable_comparison.h"
```

Comment:
We recently renamed files to avoid the duplicate "variable" -- since this is in *folder*/*module* `variable` we don't need it in the filename as well.